src/libcore/alloc.rs

```diff
@@ -1219,3 +1227,10 @@ pub unsafe trait Alloc {
         }
     }
 }
+
+// In order to rename `Alloc` to `AllocRef`, some submoduleshas to be updated as well. The CI fails
+// if either of the submodules fails to compile. The submodules have their own CI depending on a
+// specific Rust version, which don't have `AllocRef` yet. This alias is used to make the submodules
+// compile and pass the CI.
```

Comment:
AFAIK the submodules, when being tested on rustc CI, use the rustc version under testing, not some specific Rust nightly.

Also, braking Miri temporarily is entirely fine and happens all the time. I am not sure why this was added.

Reply:
They do, but they import the submodules with a specific commit, which don't have `AllocRef`. Is there a way to allow the tools CI to fail? This would come in handy for upcoming changes to the `AllocRef` trait, when it's not possible to simply reexport a trait.

Reply:
Tools CI *is* allowed to fail. See https://forge.rust-lang.org/infra/toolstate.html for details.

Reply:
Ah, then @alexcrichton probably canceled the merge because of the upcoming stable release

Reply:
> If a PR lands on `master` in the week before the beta is cut, and that PR regresses the tool (if it makes the state "worse"), CI fails.

Reply:
The nomicon is [shipped on stable](https://doc.rust-lang.org/nomicon/) to it better should have its tests pass there.